plugins/woocommerce/includes/rest-api/Controllers/Version3/class-wc-rest-coupons-controller.php

```diff
@@ -24,4 +24,21 @@ class WC_REST_Coupons_Controller extends WC_REST_Coupons_V2_Controller {
 	 * @var string
 	 */
 	protected $namespace = 'wc/v3';
+
+	protected function prepare_objects_query($request) {
+		$args = WC_REST_CRUD_Controller::prepare_objects_query( $request );
```

Comment:
Since `WC_REST_Coupons_Controller` extends `WC_REST_Coupons_V2_Controller` which extends `WC_REST_CRUD_Controller`, we can use `$this` to reference the `prepare_objects_query` method.

```suggestion
		$args = $this->prepare_objects_query( $request );
```